src/common/io-config/src/python.rs

```diff
@@ -380,9 +376,9 @@ impl S3Config {
     #[staticmethod]
     pub fn from_env(py: Python) -> PyResult<Self> {
         let io_config_from_env_func = py
-            .import(pyo3::intern!(py, "daft"))?
-            .getattr(pyo3::intern!(py, "daft"))?
-            .getattr(pyo3::intern!(py, "s3_config_from_env"))?;
```

Comment:
We should keep the use of `pyo3::intern!` since it allows us to just create the python string once per program

https://pyo3.rs/main/doc/pyo3/macro.intern#example-using-intern-to-avoid-needlessly-recreating-the-same-python-string